### ISSUE TITLE
Foraging Amplitude Units

### ISSUE THREAD

**Issue by MichaelCurrie** [openworm/open-worm-analysis-toolbox#206]
From @ver228 

> A quick question. I am documenting the features however I found in this document ( https://github.com/openworm/open-worm-analysis-toolbox/blob/master/documentation/database%20schema/Features%20Specifications.xlsx)  that locomotion.foraging_bends.amplitude should have units of micrometers.
> 
> However, for what i understand from the code the foraging_bends.amplitude is the maxima of the nose angles within a time window (https://github.com/openworm/open-worm-analysis-toolbox/blob/master/open_worm_analysis_toolbox/features/locomotion_bends.py#L679), so it should have units of degrees.
> 
> I think the units might be wrong in the original paper. From the Supplementary Material "The foraging amplitude is defined as the largest foraging angle measured, prior to crossing 0°", and because it is an angle it clearly should have units of degrees, but then in Supplementary Table 2 the corresponding features do have units of Microns.

**Comment by MichaelCurrie** [comment]
@ver228 from everything you say it certainly makes sense that it should be degrees.  I wonder if we should clarify from @Eviatar.

**Comment by Eviatar** [comment]
Yep, my mistake. We discovered this a few days ago. I've been working with Andre & Avelino to help move the code forward. There are a few things on the list & the goal is to take care of them just after the UCLA worm meeting this coming week.

**Comment by MichaelCurrie** [comment]
OK thanks, see you there btw!  WCON's poster session is Thursday 22 June from 21 to 22:30.

**Comment by Eviatar** [comment]
Awesome, looking forward to it!
